api-report/merge-tree.api.md

```diff
@@ -991,13 +936,13 @@ export class SegmentGroupCollection {
     constructor(segment: ISegment);
     // (undocumented)
     copyTo(segment: ISegment): void;
-    // (undocumented)
+    // @internal (undocumented)
```

Comment:
no need to mark members internal on an internal type

Reply:
I think the GitHub diff makes this look like it is part of `SegmentGroup`. This is a field on `SegmentGroupCollection`, which is public